R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -196,6 +196,12 @@ test_that("create DataFrame from RDD", {
   expect_equal(dtypes(df), list(c("name", "string"), c("age", "int"), c("height", "float")))
   expect_equal(as.list(collect(where(df, df$name == "John"))),
                list(name = "John", age = 19L, height = 176.5))
+  expect_equal(getNumPartitions(toRDD(df)), 1)
```

Comment:
Thanks so much for digging into this ! I kind of like the behavior of `positions` (distributing things evenly etc.) and also the fact that we will be maintaining similar behavior as scala with this new function.  And the `lapply` code fragment is not that much more complex imho in terms of readability

So my take is lets go for it ? We can also add a comment saying we are mimicing positions in case somebody wonders why we didn't use a one-liner.